ui/tests/helpers/general-selectors.ts

```diff
@@ -21,6 +21,7 @@ export const GENERAL = {
 
   filter: (name: string) => `[data-test-filter="${name}"]`,
   filterInput: '[data-test-filter-input]',
+  filterInputSearch: (attr: string) => `[data-test-input-search="${attr}"]`,
```

Comment:
nit: filter seems unnecessary here
```suggestion
  inputSearch: (attr: string) => `[data-test-input-search="${attr}"]`,
```